src/editions/advanced-migrations.md

```diff
@@ -118,6 +121,28 @@ If your documentation tests use syntax that is not supported in the new edition,
 In rare cases, you can manually set the edition for each test.
 For example, you can use the [`edition2018` annotation][rustdoc-annotation] on the triple backticks to tell `rustdoc` which edition to use.
 
+## Generated code
+
+Another area where the automated fixes cannot apply is if you have a build script which generates Rust code at compile time (see [Code generation] for an example).
+In this situation, if you end up with code that doesn't work in the next edition, you will need to manually change the build script to generate code that is compatible.
+
+## Migrating non-Cargo projects
+
+If your project is not using Cargo as a build system, it may still be possible to make use of the automated lints to assist migrating to the next build system.
+You can enable the migration lints as described above by enabling the appropriate [lint group].
+For example, you can use the `#![warn(rust_2021_compatibility)]` attribute or the `-Wrust-2021-compatibility` [CLI flag].
```

Comment:
Shall we suggest using `--force-warn`? (I realize this is not stable yet though)

Reply:
Sure, sounds good!